microprofile/health/src/test/java/io/helidon/microprofile/health/HealthMpServiceIT.java

```diff
@@ -186,6 +204,21 @@ public HealthCheckResponse call() {
         }
     }
 
+    /**
+     * A test {@link HealthCheckProvider} bean that should be discovered
+     * by the service loader and its provided start-up {@link HealthCheck}s added
+     * to the health check endpoint.
```

Comment:
general comment: startup vs start-up and health check vs health-check. I only saw the hyphenated versions twice.

Reply:
Fixed.